lib/requestwrapper.ts

```diff
@@ -66,6 +66,39 @@ function isAuthenticationError(error: any): boolean {
   return isAuthErr;
 }
 
+/**
+ * Return true if object has a specified property that is a string
+ * @private
+ * @param {Object} obj - object to look for property in
+ * @param {string} property - name of the property to look for
+ * @returns {boolean} true if property exists and is string
+ */
+function hasStringProperty(obj: any, property: string): boolean {
+  return Boolean(obj[property] && typeof obj[property] === 'string');
+}
+
+/**
+ * Look for service error message in common places, by priority
+ * first look in `errors[0].message`, then in `error`, then in
+ * `message`
+ * @private
+ * @param {Object} response - error response body received from service
+ * @returns {string | undefined} the error message if is was found, undefined otherwise
+ */
+function parseServiceErrorMessage(response: any): string | undefined {
+  let message;
+
+  if (Array.isArray(response.errors) && response.errors.length > 0 && hasStringProperty(response.errors[0], 'message')) {
+    message = response.errors[0].message;
+  } else if (hasStringProperty(response, 'error')) {
+    message = response.error;
+  } else if (hasStringProperty(response, 'message')) {
+    message = response.message;
+  }
+
+  return message;
```

Comment:
Does this fall back to a generic error if we can't find an error in these places?

Reply:
Yep! If you look at Line 124, it sets the message to `axiosError.statusText` (which is a generic message for the status code and is always defined) if that function returns `undefined`